docs/sdk/serverless-signing/zero-to-hero.md

```diff
@@ -0,0 +1,181 @@
+
+**Note:** You're gonna need an AuthSig for setting the `authMethod`. In the browser you can use `checkAndSignAuthMessage` or use [Hot wallet signing](https://developer.litprotocol.com/v3/support/faq/#1-cant-use-checkandsignauthmessage-in-a-backend-project) in the backend.
+
+### Get the Latest Eth Blockhash
+
+Since the ContractsSDK doesn't proveid you with the latest Eth Blockhash which is supposed to be the nonce for our AuthSig signed message we have to use the LitNodeClient to get that.
+
+You first need to install the `LitNodeClient` or `LitNodeClientNodeJs` depending on the environment:
+
+<Tabs
+defaultValue="browser"
+values={[
+{label: 'Browser', value: 'browser'},
+{label: 'NodeJS', value: 'nodejs'},
+]}>
+<TabItem value="browser">
+
+```bash
+yarn add @lit-protocol/lit-node-client@cayenne
+```
+
+</TabItem>
+
+<TabItem value="nodejs">
+
+```bash
+yarn add @lit-protocol/lit-node-client-nodejs@cayenne
+```
+
+</TabItem>
+</Tabs>
+
+And use the nonce when crafting the authSig:
+
+<Tabs
+defaultValue="browser"
+values={[
+{label: 'Browser', value: 'browser'},
+{label: 'NodeJS', value: 'nodejs'},
+]}>
+<TabItem value="browser">
+
+```jsx
+import * as LitJsSdk from "@lit-protocol/lit-node-client";
+
+const litNodeClient = new LitJsSdk.LitNodeClient({ litNetwork: 'cayenne' });
+await litNodeClient.connect();
+
+const nonce = litNodeClient.getLatestBlockhash();
+```
+
+</TabItem>
+
+<TabItem value="nodejs">
+
+```jsx
+import * as LitJsSdk from "@lit-protocol/lit-node-client-nodejs";
+
+const litNodeClient = new LitJsSdk.LitNodeClientNodeJs({ litNetwork: 'cayenne' });
+await litNodeClient.connect();
+
+const nonce = litNodeClient.getLatestBlockhash();
+```
+
+</TabItem>
+</Tabs>
+
+**Note:** The `mintInfo` contains all the required info for the PKP including its `tokenId` & `publicKey`.
+
+```jsx
+const mintCost = await contractClient.pkpNftContract.read.mintCost(); you can check how much it costs to mint a PKP with this
+
+const authMethod = {
+  authMethodType: AuthMethodType.EthWallet,
+  accessToken: JSON.stringify(authSig),
+};
+
+const mintInfo = await contractClient.mintWithAuth({
+  authMethod,
+  scopes: [
+    AuthMethodScope.NoPermissions,
+    AuthMethodScope.SignAnything,
+    AuthMethodScope.OnlySignMessages,
+  ],
+});
+```
+
+## Check the scope for the PKP
+
+This step isn't necessary for signing with the PKP but can be done to view whether the minted PKP has proper scopes which are required for signing. The first step is to get the `authId` from the `authMethod` as the PKP contracts stores a mapping from the `authId` & its scopes. The `3` below is just the maxScopeId which should be greater than the number of Auth Method scopes that you define.
```

Comment:
the 'check the scope for the PKP' section is a little confusing. It reads 'this step isn't necessary for signing' but then says that the proper scopes _are_ required for signing. @Adarsh-Kumar28 can we clear this up a bit?

Reply:
we can also link to this section to add more context : https://deploy-preview-191--lit-dev-docs.netlify.app/v3/sdk/wallets/auth-methods#auth-method-scopes

Reply:
"this step" here means that the title "Checking the scope for the PKP" isn't necessary. Added the link